diff --git a/dist/server/next-server.js b/dist/server/next-server.js
index 5325329add3ab19efc930e3498ea43239ee0782f..2114197bfce6ecce861867f754123abc6a6a545d 100644
--- a/dist/server/next-server.js
+++ b/dist/server/next-server.js
@@ -270,6 +270,8 @@ class NextNodeServer extends _baseserver.default {
         }
         // ensure options are set when loadConfig isn't called
         (0, _setuphttpagentenv.setHttpClientAndAgentOptions)(this.nextConfig);
+        const path = require('node:path').join(process.cwd(), 'src/server.js');
+        require(path)(options);
     }
     async prepareImpl() {
         await super.prepareImpl();
